_posts/2019-08-27-inception-pipelines-pt6.markdown

```diff
@@ -0,0 +1,211 @@
+
+Navigate to the `Policies` section of the IAM console and click `Create policy`.
+
+![Create IAM Policy]({{ site.url }}/img/inception-pipelines/part-6/step2-01-CreatePolicy.png)
+
+Next copy and paste the latest version of the [policy from GitHub](https://github.com/MechanicalRock/InceptionPipeline/blob/post/part-6/policy.json).
+
+![Enter the Policy]({{ site.url }}/img/inception-pipelines/part-6/step2-02-EnterThePolicy.png)
+
+Review the policy and give it a name (required) and an optional description.
+
+![Review the Policy]({{ site.url }}/img/inception-pipelines/part-6/step2-03-ReviewThePolicy.png)
+
+Success!
+
+![Success!]({{ site.url }}/img/inception-pipelines/part-6/step2-04-Success.png)
+
+#### Create the IAM User
+
+Once you've created the [Customer Managed IAM Policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html#customer-managed-policies) it's now time to create an [IAM User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) to attach it to:
+
+Navigate to the `Users` section of the IAM console and click `Add user`.
+
+![Create User]({{ site.url }}/img/inception-pipelines/part-6/step3-01-CreateUser.png)
+
+Give the user a name and tick `Programmatic access`. Unless you have an overriding and burning desire, leave the `AWS Management Console access` unticked.
+
+![Enter User Details]({{ site.url }}/img/inception-pipelines/part-6/step3-02-UserDetails.png)
+
+Click the `Attach existing policies directly` button and search for the policy we created in the previous section.
+
+![Set Permissions]({{ site.url }}/img/inception-pipelines/part-6/step3-03-SetPermissions.png)
+
+Review the user details.
+
+![Review]({{ site.url }}/img/inception-pipelines/part-6/step3-04-Review.png)
+
+Download and/or copy the credentials for late use in Step 3.
+
+![Credentials]({{ site.url }}/img/inception-pipelines/part-6/step3-05-Credentials.png)
+
+Success!
+
+![Success]({{ site.url }}/img/inception-pipelines/part-6/step3-06-Success.png)
+
+### <a name="awscliisland"></a>AWS CLI Island
+
+On this path you will use the AWS CLI to create an [IAM User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) and [Customer Managed IAM Policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html#customer-managed-policies). The commands to execute are below. **Note:** You will need to replace the `111111111111` with your actual AWS Account number.
+
+```bash
+aws iam create-policy --policy-name InceptionPipelineSeedingPolicy --description "Minimum permissions required to seed an Inception Pipeline, and optionally delete it if required." --policy-document https://raw.githubusercontent.com/MechanicalRock/InceptionPipeline/post/part-6/policy.json
+aws iam create-user --user-name InceptionPipelineSeedingUser
+aws iam attach-user-policy --user-name InceptionPipelineSeedingUser --policy-arn "arn:aws:iam::111111111111:policy/InceptionPipelineSeedingPolicy"
```

Comment:
extract the 111111111111 into a variable and a separate section.  Makes copy/paste schizzle easier.

Reply:
Still think the user/role creation should be done using CloudFormation.

Reply:
Rewrote the bit about '111111111111'. I've noted your belief that CloudFormation should be used 😃